tests/xilinx_test.sh

```diff
@@ -119,6 +119,9 @@ run_all() {
     run_sample blas/nodes/gemv_test gemv_FPGA_Accumulate_float_False_w4_1 1 --target accumulate --vectorize 4
     run_sample blas/nodes/ger_test ger_test_1 1 --target fpga
 
+    ## STD LibNodes
+    run_multi_sample fpga/reduce_fpga 0 "reduction_sum_one_axis" "reduction_sum_all_axis" "reduction_sum_4D" "reduction_max"
```

Comment:
In this case it's quite important to test that this has been pipelined successfully -- can you make sure it tests, e.g., sum with doubles, and checks that the partial product implementation pipelines this?